src/chainlit/element.py

```diff
@@ -1,23 +1,27 @@
 from pydantic.dataclasses import dataclass
-from dataclasses_json import dataclass_json
-from typing import Dict, Union, Any
+from typing import Dict, List, Union, Any
 import uuid
 import aiofiles
 from io import BytesIO
 
 from chainlit.context import get_emitter
-from chainlit.client import BaseClient
+from chainlit.client.base import BaseClient
 from chainlit.telemetry import trace_event
 from chainlit.types import ElementType, ElementDisplay, ElementSize
 
 type_to_mime = {
-    "image": "binary/octet-stream",
+    "image": "image/png",
```

Comment:
Not sure this is a good idea, the best would be to detect the image mime type on the server side (for example using https://stackoverflow.com/a/2753385/2831645).
I believe it will be an issue if folks try to upload an SVG.

Reply:
I agree but then on the local file system images are not displayed at all since they have no extension. This is also problematic

Reply:
I mean when a user browse the `/fs` folder on windows or mac